### ISSUE TITLE
Post install for clisp failing (no UIOP)

### ISSUE THREAD

**Issue by Vincent-de-Comarmond** [roswell/roswell#489]
## Environment:
Linux

## uname -a:
Linux vdecomarmond-G5-5590 5.8.0-63-generic #71~20.04.1-Ubuntu SMP Thu Jul 15 17:46:08 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

## ros --version:
 roswell 20.06.14.107

## ros config: 
sbcl.version=2.1.0
clisp.version=2.49
sigsegv.version=2.11
ffcall.version=ffcall-1-10
emacs.type=sly
sly.version=git
setup.time=3836093252
sbcl-bin.version=2.0.11
default.lisp=clisp

## How to reproduce the issue:

```sh
ros install clisp
ros use clisp
ros run clisp
```

## Expected behavior

Roswell runs post-install successfully.
Clisp repl runs as per usual (with uiop, quicklisp, etc.) included

## Actual result

```sh
*** - READ from
      #<INPUT BUFFERED FILE-STREAM CHARACTER #P"/usr/local/etc/roswell/extend-quicklisp.lisp" @67>:
      there is no package with name "UIOP"
The following restarts are available:
...
```

**Comment by guicho271828** [comment]
btw, how about `ros install clisp-head` on the latest roswell? Does it work?

**Comment by Vincent-de-Comarmond** [comment]
> btw, how about `ros install clisp-head` on the latest roswell? Does it work?

No luck with this

**Comment by guicho271828** [comment]
Just to check, did you run 
```ros use clisp-head``` too?

**Comment by Vincent-de-Comarmond** [comment]
> Just to check, did you run `ros use clisp-head` too?

No ... the issue persists. 
There was no clisp-head available in roswell 20.06.14.107
So I upgraded 

Currently, then
uname -a: Linux myunsername 5.11.0-41-generic #45~20.04.1-Ubuntu SMP Wed Nov 10 10:20:10 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
ros --version: roswell 21.10.14.111(8fc2c35)
ros config:
clisp-head.version=21.9.4
sbcl.version=2.1.0
clisp.version=2.49
sigsegv.version=2.11
ffcall.version=ffcall-1-10
emacs.type=sly
sly.version=git
setup.time=3848022424
sbcl-bin.version=2.0.11
default.lisp=clisp-head

Error = the same as before:
*** - READ from #<INPUT BUFFERED FILE-STREAM CHARACTER #P"/usr/local/etc/roswell/extend-quicklisp.lisp" @67>: there is no package with name "UIOP"
The following restarts are available:
RETRY          :R1      Retry compiling #<CL-SOURCE-FILE "roswell.extend.quicklisp" "extend-quicklisp">.
ACCEPT         :R2      Continue, treating compiling #<CL-SOURCE-FILE "roswell.extend.quicklisp" "extend-quicklisp"> as having been successful.
ABORT          :R3      Give up on "roswell.extend.quicklisp"
SKIP           :R4      skip (QUICKLOAD roswell.extend.quicklisp SILENT ...)
RETRY          :R5      retry (QUICKLOAD roswell.extend.quicklisp SILENT ...)
STOP           :R6      stop loading file /usr/local/etc/roswell/patch-quicklisp.lisp
SKIP           :R7      skip (INCLUDE patch-quicklisp)
RETRY          :R8      retry (INCLUDE patch-quicklisp)
STOP           :R9      stop loading file /home/vdecomarmond/.roswell/lisp/quicklisp/local-init/roswell.lisp
SKIP           :R10     Skip local setup file #P"/home/vdecomarmond/.roswell/lisp/quicklisp/local-init/roswell.lisp"
ABORT          :R11     Stop loading local setup files
SKIP           :R12     skip (SETUP)
RETRY          :R13     retry (SETUP)
STOP           :R14     stop loading file /home/vdecomarmond/.roswell/lisp/quicklisp/setup.lisp
SKIP           :R15     skip (UNLESS # # ...)
RETRY          :R16     retry (UNLESS # # ...)
STOP           :R17     stop loading file /usr/local/etc/roswell/init.lisp
Break 1 ROSWELL.UTIL[3]>

**Comment by Vincent-de-Comarmond** [comment]
Okay ... an update

The above problem disappears after `ros install asdf`
I did not expect this behaviour. Is this expected behaviour ... but maybe I'm not understanding something.

Please feel free to close this if you think there's nothing worth investigating here.
